spec/factories/ad_unit.rb

```diff
@@ -9,5 +9,8 @@
     ad_format { AdUnit::FORMAT_VIDEO }
     last_served_at nil
     active true
+    rewarded false
+    interstitial false
+    video_length 0
```

Comment:
add trait for `:video`, `:interstitial`, `:rewarded`
`https://robots.thoughtbot.com/remove-duplication-with-factorygirls-traits`